src/containers/Supplier/Table/TableSupplier.tsx

```diff
@@ -73,8 +73,9 @@ const TableSupplier = (props: handleShowType) => {
     axios.get<Data>(url, {
       params: queryParams
     }).then((response) => {
-      console.log(response.data)
-      setData(response.data)
+      setData(response.data);
+    }).catch((error) => {
+      alert('Terjadi kesalahan pada panggilan API: ' + error);
```

Comment:
dont use alert. use this https://www.npmjs.com/package/react-toastify. 
and I thing `error` is not correct message

Reply:
done, use react-toastify and change the error message